easybuild/easyconfigs/g/GATK/GATK-4.2.0.0-GCCcore-10.2.0-Java-1.8.eb

```diff
@@ -0,0 +1,53 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 Cyprus Institute / CaSToRC, University of Luxembourg / LCSB
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis.georgatos@uni.lu>,
+#             Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# Modified by: Adam Huffman, Jonas Demeulemeester
+# The Francis Crick Institute
+# Modified for version 4.0.5.1 by: Ruben van Dijk, University of Groningen
+##
+
+easyblock = 'Tarball'
+
+name = 'GATK'
+version = '4.2.0.0'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://www.broadinstitute.org/gatk/'
+description = """The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute
+ to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
+ with a primary focus on variant discovery and genotyping as well as strong emphasis on
+ data quality assurance. Its robust architecture, powerful processing engine and
+ high-performance computing features make it capable of taking on projects of any size."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/broadinstitute/gatk/releases/download/%(version)s/']
+sources = ['gatk-%(version)s.zip']
+checksums = ['dd11cc8e3bc7a23c2c226366428f0908c902765eabbc1c641c736c06b80aaf78']
+
+multi_deps = {'Python': ['3.8.6', '2.7.18']}
```

Comment:
Wouldn't we remove the Python 2 version now?

Reply:
@verdurin since Python2 is no longer maintained for >1 year now, I agree it's better to not provide it anymore (at least not by default).

removed in https://github.com/easybuilders/easybuild-easyconfigs/pull/12512/commits/1ef3e8cb4b8b8570d0d2990ab7dc093d53319ae2